marklogic-data-hub-central/ui/e2e/cypress/e2e/mastering/stepsAndRunFlowValidations.cy.tsx

```diff
@@ -0,0 +1,227 @@
+    cy.resetTestUser();
+    cy.waitForAsyncRequest();
+  });
+  it("Create a load Step", () => {
+    toolbar.getLoadToolbarIcon().click({force: true});
+    cy.waitForAsyncRequest();
+    loadPage.stepName("ingestion-step").should("be.visible");
+    loadPage.loadView("th-large").click();
+    loadPage.addNewButton("card").click();
+    loadPage.stepNameInput().clear().type(loadStepName);
+    loadPage.saveButton().click();
+    cy.findByText(loadStepName).should("be.visible");
+  });
+  it("Add load Step to New Flow", {defaultCommandTimeout: 120000}, () => {
+    loadPage.addStepToNewFlow(loadStepName);
+    cy.waitForAsyncRequest();
+    cy.findByText("New Flow").should("be.visible");
+    runPage.setFlowName(flowName1);
+    loadPage.confirmationOptions("Save").click();
+    cy.waitForAsyncRequest();
+    //Fix for Windows, if accordion it's not collapsed
+    runPage.getPatientFlowAccordion(flowName1).then(($ele) => {
+      if (!$ele.hasClass("show")) {
+        cy.log("**Toggling accordion because it was closed.**");
+        runPage.toggleExpandFlow(flowName1);
+      }
+    });
+    cy.verifyStepAddedToFlow("Loading", loadStepName, flowName1);
+    runPage.runStep(loadStepName, flowName1);
+    cy.get("input[type=\"file\"]").attachFile(["agents/agent1.json", "agents/agent2.json", "agents/agent3.json"], {force: true});
+    browsePage.waitForSpinnerToDisappear();
+    cy.waitForAsyncRequest();
+    runPage.verifyStepRunResult(loadStepName, "success");
+    runPage.closeFlowStatusModal(flowName1);
+  });
+  //add back in using storage saving with DHFPROD-8523
+  it("Create a new entity and Add properties", {defaultCommandTimeout: 120000}, () => {
+    toolbar.getModelToolbarIcon().should("be.visible").click();
+    modelPage.selectView("table");
+    entityTypeTable.waitForTableToLoad();
+    modelPage.getAddButton().click();
+    modelPage.getAddEntityTypeOption().should("be.visible").click({force: true});
+    entityTypeModal.newEntityName("Agent");
+    entityTypeModal.getAddButton().click();
+    cy.waitForAsyncRequest();
+    // Add properties
+    propertyTable.getAddPropertyButton("Agent").should("be.visible").click();
+    propertyModal.newPropertyName("Name");
+    propertyModal.openPropertyDropdown();
+    propertyModal.getTypeFromDropdown("string").click();
+    propertyModal.getSubmitButton().click();
+    propertyTable.getAddPropertyButton("Agent").should("be.visible").click();
+    propertyModal.newPropertyName("ID");
+    propertyModal.openPropertyDropdown();
+    propertyModal.getTypeFromDropdown("string").click();
+    propertyModal.getSubmitButton().click();
+    cy.wait(1000);
+    modelPage.getPublishButton().click({force: true});
+    confirmationModal.getYesButton(ConfirmationType.PublishAll);
+    toolbar.getModelToolbarIcon().should("be.visible");
+    browsePage.waitForSpinnerToDisappear();
+    cy.waitForAsyncRequest();
+  });
+  it("Create mapping step", () => {
```

Comment:
Same suggestion about the titles. They seem too generic. And there probably is another test with that name.